runtime/vara/src/weights/pallet_gear.rs

```diff
@@ -1223,6 +1228,61 @@ impl<T: frame_system::Config> pallet_gear::WeightInfo for SubstrateWeight<T> {
             // Standard Error: 8_787
             .saturating_add(Weight::from_ref_time(1_153_841).saturating_mul(r.into()))
     }
+    // FIXME: use the result of `instr_i64extendsi32` now
+    // waiting for the next generation.
+    fn instr_i32extend8s(r: u32, ) -> Weight {
+        // Proof Size summary in bytes:
+        //  Measured:  `0`
+        //  Estimated: `0`
+        // Minimum execution time: 1_900 nanoseconds.
+        Weight::from_ref_time(1_701_470)
+            // Standard Error: 2_591
+            .saturating_add(Weight::from_ref_time(312_279).saturating_mul(r.into()))
+    }
+    // FIXME: use the result of `instr_i64extendsi32` now
+    // waiting for the next generation.
+    fn instr_i32extend16s(r: u32, ) -> Weight {
+        // Proof Size summary in bytes:
+        //  Measured:  `0`
+        //  Estimated: `0`
+        // Minimum execution time: 1_900 nanoseconds.
+        Weight::from_ref_time(1_701_470)
+            // Standard Error: 2_591
+            .saturating_add(Weight::from_ref_time(312_279).saturating_mul(r.into()))
+    }
+    // FIXME: use the result of `instr_i64extendsi32` now
```

Comment:
so when would we fix it?

Reply:
@clearloop try to merge master into your branch, then run workflow: actions -> test -> branch: cl/issue-2523 without checking the checkbox. This workflow should generate the artifacts for your branch as two zip files.